web/pdf_find_controller.js

```diff
@@ -248,6 +248,15 @@ function normalize(text) {
       }
 
       if (p4) {
+        // An ideographic at the end of a line doesn't imply toadd an extra
```

Comment:
Spelling/grammar nit:
```suggestion
        // An ideographic at the end of a line doesn't imply adding an extra
```